src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      const index = Animal.alive.indexOf(this);
+
+      if (index !== -1) {
+        Animal.alive.splice(index, 1);
+      }
+    }
```

Comment:
The `checkHealth` method removes the animal from the `Animal.alive` array if its health is 0 or less. Consider adding a return statement after removing the animal to prevent unnecessary checks or operations after the animal is removed.